Main.py

```diff
@@ -0,0 +1,42 @@
+from functions import *
+
+# keyboard
+win.listen()
+win.onkeypress(rocket_left_up, "w")
+win.onkeypress(rocket_left_down, "s")
+win.onkeypress(rocket_right_up, "Up")
+win.onkeypress(rocket_right_down, "Down")
+
+score_a = 0
+score_b = 0
```

Comment:
I'm curious why you keep writing the code in a script-like fashion instead of wrapping it into a function that is executed if the file is an entry point (aka `__name__ == "__main__"`).